README.md

```diff
@@ -275,20 +619,68 @@ We can discuss what is automated, what is not, and how that can be improved. Mor
 what are the production services and software offered by HPC, and can we trace their entire pipeline
 from source code to production deployment? Is there room for improvement?
 
+> there is a lot of discussion aroudn software stacks like this right now -- we should probably
+> sync up on it. There are more reasons than just RSE-ops to have a common software stack for HPC
+> at this level.  It also helps us push on vendors.
+
 ## Summary of Opportunities
 
 The following areas of RSE-ops are not well developed, and we have opportunity to build or seek out tools to fill the space.
 
 - **Application Development**: Development framework for interactive (browser based) applications for HPC. This relies on being able to ensure secure access in a browser.
+
+> I think people are calling this "cloud services" or "web applications". If you say "app
+> development in HPC people think simulation.
```

Comment:
That's funny, it may be true! I've never heard of "app development" in HPC so it must be a national lab thing.

Reply:
it's an academic thing too -- if the academics work on an app.

Reply:
I always associated "app" with "web-app" and if there was a research app, it probably also had an interface. We will need to be careful about using this term, as it sounds like it can mean different things to different people.

Reply:
"Application" has been around since at least 1985:

* https://www.osnews.com/story/24882/the-history-of-app-and-the-demise-of-the-programmer/
* https://en.wikipedia.org/wiki/Application_software

Reply:
oh damn, that's before me! :laughing: